Models/getArtComments.models.js

```diff
@@ -0,0 +1,21 @@
+const db = require('../db/connection')
+
+
+exports.fetchArticleComments = (req) => 
+{
+
+    const articlesId = req.params.article_id
+
+
+    return db.query(`SELECT * FROM articles WHERE article_id=${articlesId}`).then((articleRows) => {
```

Comment:
This is not ok. You're opening yourself up for SQL injection completely unnecessarily. Please look into how pg can give us the ability to use parameters in our queries to stop SQL injection.

Reply:
Done.